webPageFetcher.sh

```diff
@@ -0,0 +1,524 @@
+# repository. Due to copyright problems we were unable to host the files in    #
+# the github repository and hence the creation of this program. The program    #
+# could become very useful if more web pages were added to its database, since #
+# the more choices you have the higher rate of success you will have.          #
+#                                                                              #
+################################################################################
+#******************************************************************************#
+################################################################################
+#                           EDITING THE CODE                                   #
+################################################################################
+#                                                                              #
+#                                                                              #
+################################################################################
+#******************************************************************************#
+################################################################################
+#                           START OF CODE                                      #
+################################################################################
+#******************************************************************************#
+#TODO: Editing section needs to be re-written
+set -o nounset
+
+readonly CURRENT_DIR=$(pwd)
+readonly WEB_PAGES_DIR="${CURRENT_DIR}/web-pages"
+readonly WEB_PAGES_LINKS="${CURRENT_DIR}/web-pages-links.txt"
+
+# array of all the folders in web-page directory
+FOLDERS=()
+# variable to set size of the folders array
+NUMBER_OF_FOLDERS=""
+
+# colors
+readonly RED='\033[0;31m'
+readonly GREEN='\033[0;32m'
+readonly ORANGE='\033[0;33m'
+readonly LIGHT_BLUE='\033[1;34m'
+# No Color
+readonly NC='\033[0m'
+
+###################################################
+# Name: requirment_checker                        #
+#                                                 #
+# Description: check the requirements for running #
+# the program which icludes various files and     #
+# internet.                                       #
+#                                                 #
+# Arguments: None                                 #
+#                                                 #
+# Returns: None                                   #
+#                                                 #
+# Notes: Function checks the fallowing:           #
+# web-pages-links.txt                             #
+# internet                                        #
+# wget                                            #
+# sed                                             #
+# web-page directory                              #
+###################################################
+function requirment_checker()
```

Comment:
There's no reason for this. The tool aims to work on Kali and Kali already supports all of these utilities.